docs/pyspark-migration-guide.md

```diff
@@ -19,4 +19,4 @@ license: |
   limitations under the License.
 ---
 
-The migration guide is now archived on [this page](https://spark.apache.org/docs/latest/api/python/migration_guide/index.html).
```

Comment:
I think this is correct. Please see https://github.com/apache/spark/pull/29385#discussion_r467669673.

Reply:
oh, I see.

Reply:
Yeah, currently it's a bit confusing because the redesigned doc is not released yet. Hopefully this becomes less confusing after Spark 3.1 release...